src/sql/services/notebook/notebookServiceImpl.ts

```diff
@@ -65,6 +67,17 @@ export class NotebookService implements INotebookService {
 
 		return op(provider);
 	}
+
+	//Returns an instantiation of RenderMimeRegistry class
+	getMimeRegistry(): RenderMimeRegistry
+	{
+		//let mimeRegistry: RenderMimeRegistry;
+		return new RenderMimeRegistry({
```

Comment:
Please just use 1 of these. It's minor but will set us up correctly for the future. E.g. define in the constructor, or do
```
if (!this.mimeRegistry) {
   this.mimeRegistry = new RenderMimeRegistry({
        initialFactories: standardRendererFactories
   });
}
return this.mimeRegistry;
```

Reply:
Done